src/mame/atlus/cave.cpp

```diff
@@ -21,6 +21,7 @@ Other        :  93C46 EEPROM
 Year + Game               License       PCB         Tilemaps        Sprites         Other
 -----------------------------------------------------------------------------------------
 94 Mazinger Z             Banpresto     BP943A      038 9335EX706   013 9341E7009   Z80
+95 Jumbo Godzilla         Toho          N-42 EM     038 9345E7008   013 9442WX002
```

Comment:
The rest of the entries in this table are ordered  by year, so please put this at least below Power Instinct 2